src/Algorand/AssetTransaction.cpp

```diff
@@ -0,0 +1,79 @@
+//
+// Created by Andrew Vasiliev on 26.07.2021.
+//
```

Comment:
could you please change this license header?

Reply:
Should I do it through new commit in my branch and then squash it before merge? Or I can do `git commit --amend`?